common/lib/xmodule/xmodule/course_module.py

```diff
@@ -568,7 +568,7 @@ class CourseFields(object):
     display_organization = String(
         display_name=_("Course Organization Display String"),
         help=_(
-            "Enter the course organization that you want to appear in the courseware. This setting overrides the "
+            "Enter the course organization that you want to appear in the course. This setting overrides the "
             "organization that you entered when you created the course. To use the organization that you entered "
             "when you created the course, enter null."
```

Comment:
I'm going to leave as is for now as verifying will require a larger effort. I can create a separate story to verify that the default we see is indeed what's actually saved to achieve the "null" setting.